src/auto-complete.jsx

```diff
@@ -38,11 +38,12 @@ const AutoComplete = React.createClass({
     onUpdateInput: React.PropTypes.func,
     open: React.PropTypes.bool,
     searchText: React.PropTypes.string,
-    showAllItems: React.PropTypes.bool,
```

Comment:
That looks like a breaking changed. Can we make it still work with a depreciation message. We are going to release v0.14.1 not v0.15 :grin:. You can have a look at https://github.com/callemall/material-ui/blob/master/src/left-nav.jsx#L66 for an example of how to do it.